.github/workflows/sycl-cts-on-spirv-be.yml

```diff
@@ -0,0 +1,77 @@
+# This workflow builds SYCL-CTS with -fsycl-use-spirv-backend-for-spirv-gen and
+# runs it with opencl:cpu & gen12 and level_zero:gpu & PVC on Sundays.
+
+name: SYCL-CTS with SPIR-V backend
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 7'
+
+permissions: read-all
+
+jobs:
+  ubuntu2204_build:
+    if: github.repository == 'intel/llvm'
+    uses: ./.github/workflows/sycl-linux-build.yml
+    secrets: inherit
+    with:
+      build_cache_root: "/__w/"
+      build_artifact_suffix: default
+      build_configure_extra_args: '--hip --cuda'
+      build_image: ghcr.io/intel/llvm/ubuntu2204_build:latest
```

Comment:
we can probably use ubuntu 2404

Reply:
No objections, but why do we use 2204 in regular nightly?

Reply:
because some customers use the nightly images, and theres a new glibc version in 2404 so if we build with 2404 you cant run it on 2204, so we didnt want to break users on 2204

Reply:
Removed this line, so default ubuntu2404 is used - [99b8398](https://github.com/intel/llvm/pull/16731/commits/99b8398f719b2d9c8139843024942481c3b4b578)